libs/ngx-charts-on-fhir/src/lib/fhir-mappers/observation/blood-pressure-mapper.service.ts

```diff
@@ -30,15 +29,15 @@ export class BloodPressureMapper implements Mapper<BloodPressureObservation> {
       merge({}, this.annotationOptions, {
         id: 'Systolic Blood Pressure Reference Range',
         display: true,
-        label: { content: 'Systolic Blood Pressure Reference Range' },
+        label: { content: `Systolic Blood Pressure Reference Range` },
         yScaleID: layer.datasets[0].yAxisID,
         yMax: 140,
         yMin: 90,
       }),
       merge({}, this.annotationOptions, {
         id: 'Diastolic Blood Pressure Reference Range',
         display: true,
-        label: { content: 'Diastolic Blood Pressure Reference Range' },
+        label: { content: `Diastolic Blood Pressure Reference Range` },
         yScaleID: layer.datasets[0].yAxisID,
```

Comment:
why did you change these to backticks when there are no template variables used in these strings?